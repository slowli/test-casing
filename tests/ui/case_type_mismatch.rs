use test_casing::test_casing;

#[test_casing(2, ["test", "this"])]
fn tested_function(_arg: i32) {
    // Does nothing
}

fn main() {}
