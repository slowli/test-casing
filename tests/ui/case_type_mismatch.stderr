error[E0308]: mismatched types
 --> tests/ui/case_type_mismatch.rs:4:20
  |
4 | fn tested_function(_arg: i32) {
  |    --------------- ^^^^ expected `i32`, found `&str`
  |    |
  |    arguments to this function are incorrect
  |
note: function defined here
 --> tests/ui/case_type_mismatch.rs:4:4
  |
4 | fn tested_function(_arg: i32) {
  |    ^^^^^^^^^^^^^^^ ---------
